core/io/plist.cpp

```diff
@@ -383,14 +383,14 @@ void PListNode::store_text(String &p_stream, uint8_t p_indent) const {
 			p_stream += String("\t").repeat(p_indent);
 			p_stream += "<data>\n";
 			p_stream += String("\t").repeat(p_indent);
-			p_stream += data_string + "\n";
+			p_stream += String::utf8(data_string) + "\n";
 			p_stream += String("\t").repeat(p_indent);
 			p_stream += "</data>\n";
 		} break;
 		case PList::PLNodeType::PL_NODE_TYPE_DATE: {
 			p_stream += String("\t").repeat(p_indent);
 			p_stream += "<date>";
-			p_stream += data_string;
+			p_stream += String::utf8(data_string);
```

Comment:
This is ISO 8601 date and time string (`YYYY-MM-DDTHH:MM:SS`) so it's ASCII only.

Reply:
`data_string` was parsed as `utf8` elsewhere in the file. Should I still make changes? To be consistent it should probably be changed throughout the file.

Reply:
`data_string` is used for different values, some are UTF-8 some are not, it depends on `data_type`.